provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -665,8 +693,8 @@ func buildProxy(state *ClusterRosaClassicState, builder *cmv1.ClusterBuilder) (*
 // available in the channel group
 func (r *ClusterRosaClassicResource) getAndValidateVersionInChannelGroup(ctx context.Context, state *ClusterRosaClassicState) (string, error) {
 	channelGroup := ocm.DefaultChannelGroup
-	if !state.ChannelGroup.Unknown && !state.ChannelGroup.Null {
-		channelGroup = state.ChannelGroup.Value
+	if !state.ChannelGroup.IsUnknown() && !state.ChannelGroup.IsNull() {
```

Comment:
@JohnStrunk is adding function to common to validate empty var, maybe we can push his pr and rebase with moving to this new func?
https://github.com/terraform-redhat/terraform-provider-rhcs/pull/275/files#diff-2372e7e171f3c527aaa6d5927cc12d0cdd30c13a538ec3d7e92a9375ecedaae0R187